src/client/driver/status.js

```diff
@@ -4,6 +4,7 @@ export default class DriverStatus extends Assignable {
     constructor (obj) {
         super(obj);
 
+        this.id              = Date.now();
```

Comment:
do we really need this? With the current protocol we can't have more than one instance of `DriverStatus` at the same time. We create the next status after we got the response for the previous. So technically `Date.now()` is enough.